digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/model/vips/CreateImpoundmentServiceResponse.java

```diff
@@ -0,0 +1,42 @@
+package ca.bc.gov.open.digitalformsapi.viirp.model.vips;
+
+/**
+ * 
+ * CreateImpoundmentServiceResponse
+ * 
+ * 
+ * @author smillar
+ *
+ */
+public final class CreateImpoundmentServiceResponse extends VipswsBasicResponse {
+	
+	private Long impoundmentId;
+    
+    public CreateImpoundmentServiceResponse(){};
+
+    public CreateImpoundmentServiceResponse(int respCd, String respMsg, Long impoundmentId){
+        this.respCd = respCd;
+        this.respMsg = respMsg;
+        this.impoundmentId = impoundmentId; 
+    }
```

Comment:
Because this is exactly the model that VIPS uses and is what the controller serializes the VIPS response into before passing back the actual CreateImpoundmentServiceResponse object that is defined within the specification. Just keeps things easier to deal with and we still adhere to the specification from the POV of the controller delegate.